package.json

```diff
@@ -4,11 +4,11 @@
   "description": "This is a Workshop about Cypress",
   "main": "index.js",
   "scripts": {
-    "clean:reports": "rm -f cypress//reports",
-    "test:open": "npm run lint && npm run pretest && cypress open || npm run posttest",
+    "clean:reports": "rm -f cypress/reports",
+    "test:open": "npm run lint && cypress open",
     "pretest": "npm run clean:reports",
-    "test": "cypress run",
-    "posttest": "npm run combine:reports && generate:reports",
+    "test": "cypress run || npm run posttest ",
```

Comment:
I understand with this, that you want to generate the report even if the test fails, if so, you can change the command in this way: `cypress run || true` the ` || true` changes the exit command of the test, so the posttest will be always executed. However, this is unorthodox, I don't know the smart way